packaging/common/BUILD

```diff
@@ -0,0 +1,73 @@
+nfpm_content_dirs(
+    name="root_dirs",
+    dirs=[
+        # "/etc/bash_completion.d",
+        # "/etc/logrotate.d",
```

Comment:
This is defined in https://github.com/StackStorm/st2-packages/blob/master/packages/st2/debian/st2.dirs but I don't think they're necessary. When we add the files that go in these directories, the directories will be implicitly added, and we don't have specific permissions to set on these directories as we assume they will be created by the relevant package that owns the directory.

So, I left it commented out. I can either remove them now, or we can remove them once the files that go in these directories get registered (in a follow-up PR).